plugins/parquet/README.md

```diff
@@ -77,5 +82,77 @@ To select the fields that you wish to partition your data on pass in a string of
 ```
 ParquetIO.HivePartition.Write(outDataset, rowSize, '/source/directory/partioned_dataset', overwriteOption, 'year;month;day');
 
-ParquetIO.DirectoryPartition.Read(outDataset, rowSize, '/source/directory/partioned_dataset', overwriteOption, 'year;month;day');
-```
\ No newline at end of file
+ParquetIO.DirectoryPartition.Write(outDataset, rowSize, '/source/directory/partioned_dataset', overwriteOption, 'year;month;day');
+```
+
+## Apache Parquet/Arrow to ECL Type Mappings
+### ECL Record to Arrow Schema Mappings
+
+| ECL Record Type | Apache Arrow/Parquet Type | Notes |
+|:----------------|:--------------------------|:------|
+| BOOLEAN         | Boolean                   |       |
+| INTEGER         | Int32                     | Defaults to 8 bytes (64-bit). Can be explicitly sized using INTEGER1, INTEGER2, INTEGER4, or INTEGER8 |
```

Comment:
INTEGER defaults to 8 bytes, but it still has the default type for INTEGER as Int32. Same for UNSIGNED.